pkg/kvstore/allocator/allocator_test.go

```diff
@@ -87,15 +87,17 @@ func (s *AllocatorSuite) TestSelectID(c *C) {
 
 	// allocate all available IDs
 	for i := minID; i <= maxID; i++ {
-		id, val := a.selectAvailableID()
+		id, val, id2 := a.selectAvailableID()
 		c.Assert(id, Not(Equals), NoID)
 		c.Assert(val, Equals, id.String())
+		c.Assert(id, Equals, id2)
 		a.mainCache.cache[id] = TestType(fmt.Sprintf("key-%d", i))
 	}
 
 	// we should be out of IDs
-	id, val := a.selectAvailableID()
+	id, val, id2 := a.selectAvailableID()
```

Comment:
I would call this unmaskedID instead of id2 to make it clear

Reply:
Fixed :)